tests/examples/test_examples.py

```diff
@@ -0,0 +1,91 @@
+import glob
+import os
+import subprocess
+import sys
+
+import pytest
+
+NO_EXAMPLES = "no examples found"
+
+
+def can_import_unstructured():
+    try:
+        import unstructured  # noqa: F401
+
+        return True
+    except ImportError:
+        return False
+
+
+get_started_examples = [
+    filename
+    for filename in glob.glob("examples/get_started/**/*.py", recursive=True)
+    if "torch" not in filename or os.environ.get("RUNNER_OS") != "Linux"
+]
+
+llm_and_nlp_examples = [
+    filename
+    for filename in glob.glob("examples/llm_and_nlp/**/*.py", recursive=True)
+    # no anthropic token
+    if "claude" not in filename
+    and ("unstructured" not in filename or can_import_unstructured())
+] or [NO_EXAMPLES]
```

Comment:
[F] If we cannot `import unstructured` then we have no `llm_and_nlp_examples` and the group fails on Windows with this OR statement.